content/docs/studio/troubleshooting.md

```diff
@@ -1,3 +1,12 @@
+<admon>
+
+**We have renamed Views to Projects in Iterative Studio.**
+
+Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
```

Comment:
Btw, I don' think we ever use the term "dashboard" inside the Studio itself. It is more like an internal term, but maybe we should expose it to the general public. We can name a tab "dashboard" instead of "projects", for instance.

Anyway, for now, probably it is fine as I see we use this term extensively in the docs. Though probably we should create an internal issue to discuss it with the Studio team.

Reply:
Thanks Max. Not changing this for now. Have created https://github.com/iterative/studio/issues/3909 to discuss this.